docs/API_Server.md

```diff
@@ -396,6 +396,22 @@ and might later produce asynchronous messages such as:
 `{"params":{"data":[[3292.432935, 562534, 0.067059278],
 [3292.4394937, 5625322, 0.670590639]]}}`
 
+### load_cell/dump_force
+
+This endpoint is used to subscribe to force data produced by a load_cell.
+Using this endpoint may increase Klipper's system load.
+
+A request may look like:
+`{"id": 123, "method":"load_cell/dump_force",
+"params": {"sensor": "load_cell", "response_template": {}}}`
+and might return:
+`{"id": 123,"result":{"header":["time", "force (g)", "counts", "tare_counts"]}}`
+and might later produce asynchronous messages such as:
+`{"params":{"data":[[3292.432935, 40.65, 562534, -234467]]}}`
```

Comment:
Just out of curiosity, does "tare_counts" change with each sample?

Reply:
No, it only changes when the tare command is executed.

Reply:
Okay - seems a little odd to repeat this value for every sample.  Maybe just export the information via `get_status()`?